.github/pull_request_template.md

```diff
@@ -2,22 +2,22 @@
 
 **What is this about:**
 
-**What is included in the PR:** 
 
 **How does someone test / validate:** 
 
 ## Quality Checklist
 
 - [ ] **Linked issue:** #xxx
-- [ ] **Communication:** I've discussed this with core contributors in the issue. 
 - [ ] **Tests:** Added/updated and all pass
-- [ ] **Installer:** Added/updated and all pass
-- [ ] **Localization:** All end user facing strings can be localized
-- [ ] **Docs:** Added/ updated
-- [ ] **Binaries:** Any new files are added to WXS / YML
+- [ ] **Dev docs:** Added/updated
+- [ ] **New binaries:** Added on the required places
    - [ ] No new binaries
-   - [ ] [YML for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/pipeline.user.windows.yml#L68) for new binaries
-   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries
+   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
+   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
+   - [ ] [YML for build pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects
+
+### Note
+Please ensure that all strings can be localized and the [user docs](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) are updated or you have created an issue for the user docs.
```

Comment:
```suggestion
### End-user docs
Please ensure that the [end-user documentation](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) is up to date or you have created an issue for the user docs.
```

Reply:
@jaimecbernardo
Are you okay with this phrasing? Then I going to push a commit this evening (German time).

Reply:
"for the outstanding tasks." sounds weird. How do you feel about "with the remaining tasks."?
Rest LGTM!